src/Controller/ArticlesController.php

```diff
@@ -80,6 +80,12 @@ public function textAction(Request $request, string $id, int $version = null) :
                 return true;
             });
 
+        // When NUMBER of comments > 0, text must be:
+        // '<span aria-hidden="true">NUMBER</span><span class="visuallyhidden">Open annotations (there are currently NUMBER annotations on this page).</span>'
+        // When 0 comments, text must be:
+        // "<span aria-hidden=\"true\">&#8220;</span><span class=\"visuallyhidden\">Open annotations (there are currently 0 annotations on this page).</span>
```

Comment:
Add a `data-hypothesis-annotation-count` attribute like https://github.com/elifesciences/journal/blob/59a081f4dcbe6e18823f6d4138c241c1455af03c/app/Resources/views/page.html.twig#L234-L235 and it'll be updated with JS. We need this anyway (so it's a live count), but we can ignore non-JS counts for now.